<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table</class>
 <widget class="QWidget" name="Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Inventory</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image/srcc.qrc">
    <normaloff>:/img/logo1.jpg</normaloff>:/img/logo1.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: qconicalgradient(cx:0.548913, cy:1, angle:0, stop:0 rgba(0, 0, 103, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Electronics</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Groceries</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Clothing</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_invent">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>171</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/Order.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="product_name">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="prodct_lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product_desc:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cate_lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Category:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_edit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalqty_edit">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>130</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cat_edit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add INventory</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total_qty:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="desc_edt">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>170</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pricelbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Price_edit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="manu_lab">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Manufacture:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Manufac_edit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="qtysold_lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Qty_sold:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qtysold_edit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="qtysold_lbl_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amt_sold:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amount_edit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image/srcc.qrc"/>
 </resources>
 <connections/>
</ui>
